catalog/policies/git_push.default.rego

```diff
@@ -79,33 +80,33 @@ labels := input.stack.labels
 # NOTE: procesing of stack imports is disabled in the module (var.imports_processing_enabled = false),
 # and the below rules will not be evaluated by default
 # https://www.openpolicyagent.org/docs/latest/policy-language/#comprehensions
-imports := [imp | startswith(labels[i], "import:"); imp := split(labels[i], ":")[1]]
+imports := [imp | strings.any_prefix_match(labels[i], "import:"); imp := split(labels[i], ":")[1]]
```

Comment:
For cases where there's no list given as the argument (like here, where both arguments are just strings) I think there's no speedup from this function and the standard startswith might even be faster.